### DISCUSSION TITLE
这位业主是被喝茶了吗？

### DISCUSSION THREAD

**Discussion by TALK1949** [WeNeedHome/SummaryOfLoanSuspension#1036] [General]
https://github.com/WeNeedHome/SummaryOfLoanSuspension/discussions/1002#discussion-4243086

![image](https://user-images.githubusercontent.com/39082705/181306229-56168e78-b58d-40c6-835a-ffa6c2b89f08.png)

之前的标题“江苏省无锡市天渝骄园烂尾”。
因为特别关注过，我在我的浏览器历史记录里找到了原始的标题，确定是改过了。
里面还有停贷通知书和几部视频的网盘链接，现在都没了，标题内容都改了，链接也删了，啥情况？

业主们发布信息的时候最好也注意一下不要暴露过多的个人信息。

**Comment by jingzz1** [comment]
[无锡的烂尾楼业主在抗议](https://twitter.com/i/status/1551677512131887104)
